internal/callbacker/sender.go

```diff
@@ -91,24 +89,19 @@ func (p *CallbackSender) Health() error {
 }
 
 func (p *CallbackSender) Send(url, token string, dto *Callback) {
-	p.wg.Add(1)
-	go func() {
-		defer p.wg.Done()
-
-		ok := p.sendCallbackWithRetries(url, token, dto)
-
-		if ok {
-			p.updateSuccessStats(dto.TxStatus)
-		} else {
-			p.logger.Warn("Couldn't send transaction callback after retries",
-				slog.String("url", url),
-				slog.String("token", token),
-				slog.String("hash", dto.Txid),
-				slog.Int("retries", retries))
-
-			p.stats.callbackFailedCount.Inc()
-		}
-	}()
+	ok := p.sendCallbackWithRetries(url, token, dto)
+
+	if ok {
+		p.updateSuccessStats(dto.TxStatus)
```

Comment:
nit: if you return here you could remove `else { ... }`

Reply:
Done